cloud-director/v27.0.0/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ replacements:
     fieldPath: metadata.name
     options:
       delimiter: "-"
-      index: 1
```

Comment:
Can we include a comment as to why this specific provider is different to the others. Just to avoid some copy&paste mistake in the future

Reply:
Sure, will do!